ssc/cmod_fresnel_physical_iph.cpp

```diff
@@ -153,7 +153,17 @@ static var_info _cm_vtab_fresnel_physical_iph[] = {
 
     // Heat Sink
 
-    {SSC_INPUT,     SSC_NUMBER,         "pb_pump_coef",                "Pumping power to move 1kg of HTF through PB loop",                                      "kW/kg",               "",                              "Heat Sink",           "*",                "",                 ""},
+    { SSC_INPUT,    SSC_NUMBER,         "pb_pump_coef",                "Pumping power to move 1kg of HTF through PB loop",                                      "kW/kg",               "",                             "Heat Sink",            "*",                "",                 "" },
```

Comment:
This value is pre-existing and has a value in all default configurations - correct?

Reply:
That is correct, no changes were made to 'pb_pump_coef'